content/en/docs/hertz/tutorials/basic-feature/stream.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Stream"
+date: 2022-06-21
+weight: 5
+description: >
+
+---
+
+Hertz supports both server and client stream processing to improve the usability of the framework.
+
+## Server
+
+Open streaming:
+
+```go
+import (
+    "github.com/cloudwego/hertz/pkg/app/server"
+    "github.com/cloudwego/hertz/pkg/network/standard"
+    )
+
+func main() {
+    h := server.New(
+        server.WithStreamBody(),
+        server.WithTransport(standard.NewTransporter),
+    )
+    ...
+}
+```
+
+Due to different trigger modes of *netpoll* and *go net*, netpoll streams are "pseudo" streams (Due to Level Triggered in epoll, the network library will read the data into the buffer of the network library). In the case of processing large data packets (eg: updating files), there may be memory problem when using netpoll. So we recommend using the *go net* method above when in the case of processing large data packets.
+
+## Client
+
+Open streaming
+
+```go
+c, err := client.NewClient(client.WithResponseBodyStream(true))
+```
+
+Since the client has the problem of multiplexing connections, if streaming is used, the connection will be handled by the user(`resp.BodyStream()` is encapsulated by connection). There are some differences in the management of connections in the above case:
+
+1. If the user doesn't close the connection, the connection will eventually be closed by the GC without causing a connection leak. However, due to the need to wait for 2 Round-Trip Time to close the connection, in the case of high concurrency, it may happen that too many open files and cannot create a new connection.
```

Comment:
the consequence is that there will be too many open files and creating a new connection will be impossible.